index.bs

```diff
@@ -898,33 +902,37 @@ The following example shows an [[#item-resource]] in the [[#ttl-representation]]
 ## Certificate transparency ## {#using-certificate-transparency}
   Certificate Transparency [[RFC6962]] is one of a number of possible methods of proving the integrity of a register.
 
+### Entry hash ### {#entry-hash}
+
+The entry hash is the application of a supported hashing algorithm on the data contained in the [[#entry-resource]]. The [[#item-hash-datatype]] of the entry, rather than the item itself ([[#item-resource]]), is part of the raw data hashed to create the entry hash.
```

Comment:
> The [[#item-hash-datatype]] of the entry, rather than the item itself ([[#item-resource]]), is part of the raw data hashed to create the entry hash.

This is just reiterating the definition of an entry.